optaplanner-distribution/pom.xml

```diff
@@ -45,33 +45,6 @@
                 <dependencySourceInclude>org.optaplanner:optaplanner-test</dependencySourceInclude>
                 <!-- TODO apparently optaplanner-examples gets included too somehow? -->
               </dependencySourceIncludes>
-              <additionalDependencies>
```

Comment:
Some of the version properties are no longer defined as we rely on the imported `quarkus-bom`. Unfortunately, the `additionalDependencies` do not get the versions. Moving these to project dependencies.